lib/Software/Security/Policy/Individual.pm

```diff
@@ -50,6 +50,22 @@ security policy class.  Valid arguments are:
 
 the current maintainer for the distribution; B<Required>
 
+if a security_contact is defined it will override the maintainer.
```

Comment:
This text is maybe a bit ambiguous? I'm wondering it may be useful to offer answers to the question "What happens if I don't set this value?".

> If the `security_contact` field is defined, it will be presented as the primary contact point for security issues. The `maintainer` field will remain for as a contact point for other issues. If unset, the `maintainer` remains presented as the primary contact point for all issues.

Same would go for `report_url`.